pkg/detector/ospkg/bottlerocket/bottlerocket.go

```diff
@@ -0,0 +1,27 @@
+package bottlerocket
+
+import (
+	"context"
+
+	osver "github.com/aquasecurity/trivy/pkg/detector/ospkg/version"
+	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/types"
+)
+
+// Scanner implements the Bottlerocket scanner
+type Scanner struct {
+}
+
+// NewScanner is the factory method for Scanner
+func NewScanner() *Scanner {
+	return &Scanner{}
+}
+
+// Detect vulnerabilities in package using Bottlerocket scanner
```

Comment:
This is incorrect, because we don't detect vulns for Bottlerocket pkgs.

Reply:
Done.